test/spec/com/cinergix/mapper/objectmapper/GetFieldColumnMappingTest.java

```diff
@@ -0,0 +1,208 @@
+	}
+	
+	private void dropMockTableManager(){
+		dbHelper.updateData( "DROP TABLE manager" );
+	}
+	
+	/****************************************************************************************
+	 * Test getFieldColumnMapping method													*
+	 ****************************************************************************************/
+	
+	@Test
+	public void itShouldReturnNullIfGivenResultSetIsNull(){
+			
+			HashMap<Field, String> fieldsVsColumns = this.invokeCheckColumnLabelExist( SimpleUserMock.class, null );
+			
+			assertNull( "getFieldColumnMapping should return null if given ResultSet is null", fieldsVsColumns );
+			
+	}
+	
+	@Test
+	public void itShouldReturnNullIfGivenClassTypeIsNull(){
+		
+		ResultSet result = dbHelper.getResultSetForQuery( "SELECT id as user_id, name as user_name, email as user_email, age as user_age, weight as user_weight, dob as user_dob, last_update as user_last_update FROM user WHERE id LIKE 'testID'" );
+		
+		HashMap<Field, String> fieldsVsColumns = this.invokeCheckColumnLabelExist( null, result );
+		
+		assertNull( "getFieldColumnMapping should return null if given class type is null", fieldsVsColumns );
+			
+	}
+	
+	@Test
+	public void itShouldReturnNullIfBothInputPatametersAreNull(){
+		
+		HashMap<Field, String> fieldsVsColumns = this.invokeCheckColumnLabelExist( null, null );
+		
+		assertNull( "getFieldColumnMapping should return null if both input parameters are null", fieldsVsColumns );
+			
+	}
+	
+	@Test
+	public void itShouldReturnMappedSetOfFieldVsColumnNameFromTheAnnotation(){
+			
+		ResultSet result = dbHelper.getResultSetForQuery( "SELECT id as user_id, name as user_name, email as user_email, age as user_age, weight as user_weight, dob as user_dob, last_update as user_last_update FROM user WHERE id LIKE 'testID'" );
+		
+		HashMap<Field, String> fieldsVsColumns = this.invokeCheckColumnLabelExist( SimpleUserMock.class, result );
+		
+		assertTrue("getFieldColumnMapping should return a mapped set of field Vs Column names( which are in the annotation )", ( "user_name" ).equals( fieldsVsColumns.get( getField( SimpleUserMock.class, "name" ) ) ) );
+			
+	}
+	
+	@Test
+	public void itShouldReturnMappedSetOfFieldVsColumnNameFromTheAnnotationOnlyForAnnotatedFields(){
+			
+		ResultSet result = dbHelper.getResultSetForQuery( "SELECT id as user_id, name as user_name, email as user_email, age as user_age, weight as user_weight, dob as user_dob, last_update as user_last_update FROM user WHERE id LIKE 'testID'" );
+		
+		HashMap<Field, String> fieldsVsColumns = this.invokeCheckColumnLabelExist( SimpleUserMock.class, result );
+		
+		assertEquals( "getFieldColumnMapping should return a mapped set of field Vs Column names( which are in the annotation )", 1, fieldsVsColumns.size() );
+		assertTrue( "getFieldColumnMapping should return a mapped set of field Vs Column names( which are in the annotation )", ( "user_name" ).equals( fieldsVsColumns.get( getField( SimpleUserMock.class, "name" ) ) ) );
```

Comment:
In all tests each assertion should have a different message specific to the expectation.